### PR TITLE
chore: prepare for armadillo 3 release

### PR DESCRIPTION
**Pull request by marikaris**

### CODE REVIEW

**Review by erikzwart — APPROVED** [review]

File: README.Rmd [hunk 1]
```diff
@@ -38,30 +38,60 @@ methods on the data.
...
 armadillo.delete_table("project", "folder", "iris")
 armadillo.delete_project("project")
 ```
 
 ## Documentation
-For more in depth documentation please check the [howto](https://molgenis.github.io/molgenis-r-armadillo/articles/MolgenisArmadillo.html).
\ No newline at end of file
+For more in depth documentation please check the [howto](https://molgenis.github.io/molgenis-r-armadillo/articles/MolgenisArmadillo.html).
+
+## Armadillo 2
```
Comment (christianhilbrands) [inline README.Rmd:68]: I miss information about how to use it in Armadillo 3?

File: README.md [hunk 2]
```diff
@@ -45,39 +44,63 @@ them for analysis.
...
 armadillo.delete_project("project")
-#> Deleted project 'project'
 ```
 
 ## Documentation
 
 For more in depth documentation please check the
 [howto](https://molgenis.github.io/molgenis-r-armadillo/articles/MolgenisArmadillo.html).
 
+## Armadillo 2
```
Comment (christianhilbrands) [inline README.md:69]: i miss information how to use it in armadillo 3?

**Review by christianhilbrands — CHANGES_REQUESTED** [review]
RfC

**Review by christianhilbrands — APPROVED** [review]
LGTM

**Pull request merged**